src/expression.ts

```diff
@@ -815,7 +818,10 @@ export class TemplateSpan<
   }
 }
 
-export class TemplateMiddle extends BaseNode<NodeKind.TemplateMiddle> {
+export class TemplateMiddle extends BaseNode<
+  NodeKind.TemplateMiddle,
+  TemplateSpan
+> {
```

Comment:
Default types don't work anymore?

Reply:
Trying to reduce the complexity by specifying types were possible. TemplateMiddle can only be in a TemplateSpan

Reply:
Oh right - good point. This is more precise.